lib/game.rb

```diff
@@ -0,0 +1,17 @@
+
+class Game
+  GAME_RULES ={
```

Comment:
I really like how you've set up the Game class - the GAME_RULES hash means your methods are succinct and super clear - also means it has room to grow without it becoming too complicated if you wanted to add complexity to the rules (e.g. Rock, Paper, Scissors, Spock, Lizard)